packages/opentelemetry-node/src/NodeTracerProvider.ts

```diff
@@ -37,4 +45,13 @@ export class NodeTracerProvider extends BasicTracerProvider {
   stop() {
     this._pluginLoader.unload();
   }
+
+  register(config: SDKRegistrationConfig = {}) {
+    if (config.contextManager === undefined) {
```

Comment:
should you test here if `config.contextManager instanceof ScopeManager` instead ?

Reply:
`ScopeManager` is just an interface, no constructor therefore it can't be used in `instanceof` checks. This would require a real base class which is not that javascripty.

Reply:
you right I was thinking of using class but we have interface only :/

Reply:
I think that it's better to have an interface if we want to provide custom implementation. I'm not recommend instanceof <class> WDYT ?